src/js/environment/os-version.ts

```diff
@@ -31,13 +29,13 @@ export function osVersion(osEnvironment: GenericObject, agent: string): GenericO
                 break;
         }
     } else if (osEnvironment.android) {
-        version = execResult(/Android ([._\d]+)/.exec(agent), 1);
+        version = execResult(/Android ([._\d]+)/, agent);
     } else if (osEnvironment.iOS) {
-        version = execResult(/OS ([._\d]+)/.exec(agent), 1);
+        version = execResult(/OS ([._\d]+)/, agent);
     } else if (osEnvironment.mac) {
-        version = execResult(/Mac OS X (10[._\d]+)/.exec(agent), 1);
+        version = execResult(/Mac OS X ([._\d]+)/, agent);
```

Comment:
Still says OS X in pattern match. Does that still appear for Big Sur?

Reply:
yes. You can Google it
```
navigator.userAgent
"Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36"
```

Reply:
Some browsers still report major 10 because this is probably common in js libraries. That doesn't really matter. The important thing is that we parse what's there.